cli/data.go

```diff
@@ -601,118 +656,151 @@ func filenameForDownload(meta *datapb.BinaryMetadata) string {
 	return fileName
 }
 
-// tabularData downloads binary data matching filter to dst.
-func (c *viamClient) tabularData(dst string, filter *datapb.Filter, limit uint) error {
+// tabularData downloads unified tabular data and metadata for the requested data source and interval to the specified destination.
+func (c *viamClient) tabularData(dest string, request *datapb.ExportTabularDataRequest) error {
```

Comment:
Just so make sure I'm understanding this section properly, you have a go func so that we can fetch while we're still writing previous results to a file right?

Reply:
Do you mean on line 699?

Reply:
Yes

Reply:
Then yes! So we can continue getting results while writing is in progress